.github/workflows/develop.yml

```diff
@@ -170,14 +163,7 @@ jobs:
         working-directory: tests-integration/fixtures/kafka
         run: docker compose -f docker-compose-standalone.yml up -d --wait
       - name: Run sqlness
-        run: ./bins/sqlness-runner -w kafka -k 127.0.0.1:9092 -c ./tests/cases --bins-dir ./bins
-      - name: Upload sqlness logs
```

Comment:
Nothing to upload. At lease it requires fixing.

Setting `RUST_BACKTRACE=1` is useful and we may find a way to print failed cases at the very last, or as the workflow summary / annotation.